tests/tests_pytorch/utilities/test_exceptions.py

```diff
@@ -14,54 +14,31 @@
 from pathlib import Path
 
 from lightning.pytorch.utilities.exceptions import _augment_message
-from lightning.pytorch.utilities.imports import _PYTHON_GREATER_EQUAL_3_11_0
 
 
 def test_augment_message():
     # exception without args
     exception = Exception()
     _augment_message(exception, "", "new message")
     assert not exception.args
-    if _PYTHON_GREATER_EQUAL_3_11_0:
-        assert not exception.__notes__
```

Comment:
I forgot to adjust this test in #18399. I didn't notice because the CI wasn't running 3.11 yet.

Reply:
you can do it here :innocent: